### ISSUE TITLE
html_token: Processing instruction nodes are converted to text

### ISSUE THREAD

**Issue by Mr0grog** [edgi-govdata-archiving/web-monitoring-processing#149]
It looks like the differ is accidentally converting XML processing instruction nodes to renderable text. Not sure if this is a problem in the diff tokenizer or in LXML’s parsing.

http://localhost:3001/page/95bec194-cf02-407b-af1a-e286068dbe4e/b02b7003-14ad-4226-9ef6-57996d0c4ed2..10e299b1-fa3a-44ef-b121-a40de2b54805 is a good example.

The markup:

```html
<?xml encoding="UTF-8"?>
```

is converted to

```html
&lt;
<cyfunction at="" processinginstruction="">
 encoding="UTF-8"&gt;encoding="UTF-8"?
 <cyfunction at="" processinginstruction="">
  &gt;
```

<img width="599" alt="screen shot 2018-01-17 at 11 54 11 pm" src="https://user-images.githubusercontent.com/74178/35086551-c07b7dfc-fbe1-11e7-95d7-6cc08a41dd7f.png">

See the live page for how this should actually render (not at all): https://science.nasa.gov/earth-science/oceanography/physical-ocean/salinity

**Comment by Mr0grog** [comment]
Ugh, I can’t believe I haven’t gotten to this yet. This really does need fixing.

**Comment by Mr0grog** [comment]
This was fixed by switching to html5-parser in #410.
